src/test/java/org/openstreetmap/atlas/geography/atlas/dynamic/DynamicAtlasMovingTooFastTest.java

```diff
@@ -0,0 +1,84 @@
+package org.openstreetmap.atlas.geography.atlas.dynamic;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Supplier;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.geography.Rectangle;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.dynamic.policy.DynamicAtlasPolicy;
+import org.openstreetmap.atlas.geography.atlas.dynamic.rules.DynamicAtlasMovingTooFastTestRule;
+import org.openstreetmap.atlas.geography.atlas.items.Edge;
+import org.openstreetmap.atlas.geography.sharding.Shard;
+import org.openstreetmap.atlas.geography.sharding.SlippyTile;
+import org.openstreetmap.atlas.geography.sharding.SlippyTileSharding;
+import org.openstreetmap.atlas.utilities.collections.Iterables;
+
+/**
+ * @author matthieun
+ */
+public class DynamicAtlasMovingTooFastTest
```

Comment:
You gave a really good explanation of what moving too fast usually means. Can you add it here? It would give more clarity to what's being tested.